.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
```

Comment:
Can you please add Travis-CI badge to the README file?

Reply:
added